src/api/app/views/webui/package/_buildstatus.html.haml

```diff
@@ -40,7 +40,9 @@
                 = result.architecture
             .build-state.toggle-build-info-parent
               %i.fa.fa-question-circle.text-info.px-2.pl-lg-1.toggle-build-info{ title: 'Click to keep it open' }
-              = arch_repo_table_cell(result.repository, result.architecture, package, 'code' => result.code, 'details' => result.details)
+              = render(BuildresultStatusLinkComponent.new(repository_name: result.repository, architecture_name: result.architecture,
+                                                          project_name: project.name, package_name: package,
+                                                          build_status: result.code, build_details: result.details))
```

Comment:
Inside the component, `build_details` is only used to check if the details are present. [See line 19 of buildresult_status_link_component.rb](https://github.com/openSUSE/open-build-service/pull/13361/files#diff-8c4bdda2952f00c540f46af6859769c095165f9f696de5bfdb65a25d9024cc34R19).

What about passing a boolean instead?

Something like

```
= render(BuildresultStatusLinkComponent.new(repository: result.repository, architecture: result.architecture,
                                                              project: project, package_name: package,
                                                              build_status: result.code, contain_build_details: result.details.present?))
```

Reply:
I would be against this change, since this logic will have to be duplicated whenever we call this component. If the logic somehow changes, it needs to be adapted in multiple places instead of only one.

Reply:
Alright